.storybook/manager.js

```diff
@@ -2,5 +2,5 @@ import { addons } from '@storybook/addons';
 import { themes } from '@storybook/theming';
 
 addons.setConfig({
-    theme: themes.dark,
-});
\ No newline at end of file
+  theme: themes.normal,
```

Comment:
That's suspicious ( ? )

Reply:
Hahahahahahaha just to be able to watch fullscreen loader in storybook